andela/mrm-mobile
# #159987439 feature(cancel event)cancel meeting when not checked in within 10 minutes

## What does this PR do?
cancel meeting when not checked in within 10 minutes.

## Description of Task to be completed?
- [x] update countdown timer
- [x] update calendar repository to enable cancellation of event
- [x] add a task runner to carry out background checks on the check in button
- [x] remove duplicate and unnecessary codes

## How should this be manually tested?
- lunch app with the device emulator
- don't click on the login button
- check your calender to confirm event has been removed

## What are the relevant pivotal tracker stories?
159987439 

## Screenshots (if appropriate)?
N/A
